importexport/guitarpro/importgtp.cpp

```diff
@@ -1974,6 +1974,8 @@ bool GuitarPro1::readNote(int string, Note* note)
       // dead note represented as high numbers - fix to zero
       if (fretNumber > 99 || fretNumber == -1)
             fretNumber = 0;
+      if (fretNumber > 0 && string == 4)
+          fretNumber = staff->part()->instrument()->stringData()->adjustBanjo5thFret(fretNumber);
```

Comment:
How does this work on any plucked string instrument with more than 4 strings that is not a Banjo?

Reply:
adjustBanjo5thFret() makes sure number of strings == 5 and uses 'startFret' which is only set for 5-string banjos.